ITISFoundation/osparc-simcore
# is288/osparc gui

## What do these changes do?


* Re-added transitions in sidebar
* Input settings restyled
* Added drop highlighting

## Related issue number


connects to #288 

## How to test

* Checkout the branch and play with the sidebar collapse transitions to see that everything runs smoothly and has no rendering problems. Let me know if you see anything strange.
* Go to some input settings and take a look at the modifications. Is it still too difficult to understand? Do you have any suggestion? I have other improvements in mind...

BEFORE
![screenshot from 2019-03-08 14-28-51](https://user-images.githubusercontent.com/4764217/54031276-9c113880-41ae-11e9-8b08-9c9fb7f7ced8.png)
AFTER
![screenshot from 2019-03-08 14-28-11](https://user-images.githubusercontent.com/4764217/54031289-a29fb000-41ae-11e9-9340-89f7e5f4deeb.png)

[waffle.io]:https://waffle.io/marketing-assets/documents/waffleio_cheatsheet_v1.pdf?utm_source=blog&utm_medium=cheatsheet-ctabutton&utm_campaign=cheatsheet
